### DISCUSSION TITLE
How do you disable email notifications for Gist comments?

### DISCUSSION THREAD

**Discussion by mybearworld** [community/community#60097] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have my E-Mail preferences set to not give E-Mail notifications for comments and similar, but I still get notifications when I get comments on a Gist. How do I disable this?

**Comment by nirav-gajera** [comment]
-Select "Settings" from the dropdown menu.
-In the left sidebar, click on "Notifications."

-Review Notification Settings:

On the Notifications page, you will see various categories for notifications.
Scroll down to the "Gists" section.

-Adjust Gist Notification Settings:

In the "Gists" section, review the notification settings related to Gist comments.
Make sure the "Email" option is disabled or unchecked for actions that you don't want to receive email notifications.

**Reply by mybearworld** [reply]
I can't find the word Gist at all in the notification settings: https://github.com/settings/notifications
Am I in the wrong place?

**Reply by nirav-gajera** [reply]
go to Email>Email Preference ,select : Only receive account related emails, and those I subscribe to.

**Reply by mybearworld** [reply]
That I already have selected.

**Comment by nirav-gajera** [comment]
else - Go to the Gist page that you are receiving notifications for.
-Click on the "Unsubscribe" button located at the top-right corner of the Gist page, just below the user's profile image.

**Reply by mybearworld** [reply]
I'd like to still get notifications on GitHub, though.

**Comment by electricsteve** [comment]
I want an answer to this too, I was having a conversation with someone on gist and kept closing the tab after i sent a message, after the conversation I checked my email and it was full with github gist spam, please make a setting to only get notifs on github, not email.

**Comment by LabQii** [comment]
Go to your GitHub account settings to disable email notifications for Gist comments. Find the "Email notifications" section under "Notifications." Here, you can select which notifications, including Gist comments, you wish to receive. To stop receiving email notifications for Gist comments, simply uncheck the relevant boxes.

**Reply by mybearworld** [reply]
There is no "Email notifications" section.

**Reply by electricsteve** [reply]
Same

**Reply by electricsteve** [reply]
![image](https://github.com/user-attachments/assets/1585efd7-5423-49f3-a144-5cd6a9f3f3c6)

**Reply by regs01** [reply]
There is no Gist comments in Notifications

**Comment by brian6932** [comment]
I've been wondering the same thing. Randomly 2 years ago, GitHub just decided to remove gists from counting in the `Watching` category on https://github.com/settings/notifications.

**Reply by brian6932** [reply]
This seems to have been fixed around the end of September-October 2025.
